data/json/items/generic/dining_kitchen.json

```
@@ -1118,6 +1118,7 @@
     "volume": "25400 ml",
     "longest_side": "45 cm",
     "price": "68 USD",
+    "price_postapoc": "8 USD 15 cent"
```

```suggestion
    "price_postapoc": "8 USD 15 cent",
```
Just doing this to reset test suite